packages/contracts-bedrock/scripts/L2Genesis.s.sol

```diff
@@ -0,0 +1,381 @@
+        vm.etch(address(token), "");
+        vm.resetNonce(address(token));
+    }
+
+    /// @notice This contract has no initializer.
+    function _setL1Block() internal {
+        address impl = _predeployToCodeNamespace(Predeploys.L1_BLOCK_ATTRIBUTES);
+        vm.etch(
+            impl,
+            vm.getDeployedCode("L1Block.sol:L1Block")
+        );
+
+        /// Post Checks
+        _verifyProxyImplementationAddress(Predeploys.L1_BLOCK_ATTRIBUTES, impl);
+    }
+
+    /// @dev Returns true if the adress is not proxied.
+    function _notProxied(address _addr) internal pure returns (bool) {
+        return _addr == Predeploys.GOVERNANCE_TOKEN || _addr == Predeploys.WETH9;
+    }
+
+    /// @dev Returns true if the address is a predeploy.
+    function _isDefinedPredeploy(address _addr) internal pure returns (bool) {
+        return _addr == Predeploys.L2_TO_L1_MESSAGE_PASSER || _addr == Predeploys.L2_CROSS_DOMAIN_MESSENGER
+            || _addr == Predeploys.L2_STANDARD_BRIDGE || _addr == Predeploys.L2_ERC721_BRIDGE
+            || _addr == Predeploys.SEQUENCER_FEE_WALLET || _addr == Predeploys.OPTIMISM_MINTABLE_ERC20_FACTORY
+            || _addr == Predeploys.OPTIMISM_MINTABLE_ERC721_FACTORY || _addr == Predeploys.L1_BLOCK_ATTRIBUTES
+            || _addr == Predeploys.GAS_PRICE_ORACLE || _addr == Predeploys.DEPLOYER_WHITELIST || _addr == Predeploys.WETH9
+            || _addr == Predeploys.L1_BLOCK_NUMBER || _addr == Predeploys.LEGACY_MESSAGE_PASSER
+            || _addr == Predeploys.PROXY_ADMIN || _addr == Predeploys.BASE_FEE_VAULT || _addr == Predeploys.L1_FEE_VAULT
+            || _addr == Predeploys.GOVERNANCE_TOKEN || _addr == Predeploys.SCHEMA_REGISTRY || _addr == Predeploys.EAS;
+    }
+
+    /// @dev Function to compute the expected address of the predeploy implementation
+    ///      in the genesis state.
+    function _predeployToCodeNamespace(address _addr) internal pure returns (address) {
+        return address(
+            uint160(uint256(uint160(_addr)) & 0xffff | uint256(uint160(0xc0D3C0d3C0d3C0D3c0d3C0d3c0D3C0d3c0d30000)))
+        );
+    }
+
+    function _verifyProxyImplementationAddress(address _proxy, address _impl) internal view {
+        require(
+            vm.load(_proxy, PROXY_IMPLEMENTATION_ADDRESS) == bytes32(uint256(uint160(_impl))),
+            "Expected different address at Proxys PROXY_IMPLEMENTATION_ADDRESS storage slot"
+        );
+    }
+
+    function _verifyCantReinitialize(address _contract, address _arg) internal {
+        vm.expectRevert();
+        IInitializable(_contract).initialize(_arg);
+    }
+
+    function _overwriteFile(string memory _target, string memory _source) internal {
+        string[] memory commands = new string[](3);
+        commands[0] = "bash";
+        commands[1] = "-c";
+        commands[2] = string.concat(
+            "mv ",
+            _source,
+            " ",
+            _target
+        );
+        vm.ffi(commands);
```

Comment:
We can use this 1 liner to remove the need for a tempfile

```
cat <<< $(jq 'QUERY' sample.json) > sample.json
```

We just need to replace the `'QUERY'` with our desired query

Reply:
Updated! Also simplified the syntax to perform sorting